.github/workflows/stale.yml

```diff
@@ -24,3 +25,18 @@ jobs:
           exempt-all-issue-assignees: true
           operations-per-run: 300
           close-issue-reason: 'not_planned'
+
+  remove_stale: # trigger "stale" removal immediately when stale issues are commented on
+    if: github.event_name == 'issue_comment'
+    permissions:
+      contents: read #  for actions/checkout
+      issues: write #  to edit issues label
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.4.0
+      - name: Remove 'stale' label
+        run: gh issue edit ${{ github.event.issue.number }} --remove-label 'stale'
```

Comment:
Can you pull out the label in to a workflow variable and use that here on this line and in the `stale-issue-label` on line 23?

This will prevent changing either this line or line 23 in the future and neglecting to change both of them. 

https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow